eng/deploy/azure/README.md

```diff
@@ -63,6 +69,43 @@ az container create --resource-group $resourceGroup -n ml-api `
     ALLOW__EXT_PROPERTY=true MONGO_URI=$mongoUri
 ```
 
+### Enable Logging
+
+To save the logs to a file, for a summarized result, set the flag `SAVE_LOG_TO_FILE` to true, which will create a `meadowlark.log` file with the logs.
+
+For a production deployment, it's recommended to send the logs to _Log Analytics_, with the following steps:
+
+```Shell
+  # Create workspace
+  az monitor log-analytics workspace create `
+  --resource-group $resourceGroup --workspace-name meadowlark-logs
+```
+
+Copy the `customerId` from the result.
+
+```Shell
+  # Get shared key
+  az monitor log-analytics workspace get-shared-keys `
+  --resource-group $resourceGroup --name meadowlark-logs
+```
+
+Copy the `primarySharedKey` from the result.
+
+> **Note** This information can be retrieved from the Portal, in Log Analytics Workspaces -> Settings -> Agents. [More
+> information](https://learn.microsoft.com/en-us/azure/container-instances/container-instances-log-analytics#get-log-analytics-credentials)
+
+When creating the **meadowlark container**, include the customerId (workspace_id) and the primarySharedKey (workspace_key) as
+additional properties with the following flags:
+
+```Shell
+az container create ... `
+    --log-analytics-workspace <WORKSPACE_ID> `
+    --log-analytics-workspace-key <WORKSPACE_KEY>
+```
```

Comment:
How does this setup know from where to read the log data?

Reply:
This will redirect all standard output and standard error messages to Log Analytics